tck/src/main/resources/tests/updateItemPowerLevelPersentage/input.graphql

```diff
@@ -0,0 +1,8 @@
+# Tests that the implementation can handle a transformation on a primitive scalar (int).
+mutation alterIronManSuitPower {
+  updateItemPowerLevelPersentage(itemID:1001,powerLevel:"80%") {
```

Comment:
Percentage vs Persentage ?  Maybe a British English vs South African English vs American English thing?  No need to fix - just curious.

Reply:
No sorry, that is just my Afrikaans coming through. ([Persentasie](https://af.wikipedia.org/wiki/Persentasie))  . Fixed